doc/R.nvim.txt

```diff
@@ -924,10 +924,11 @@ Note: The options of this section are ignored on macOS, where the command
 `open` is called to run the default application used to run shell scripts.
 
 R will run in Neovim's built-in terminal emulator, unless you set the value of
-`external_term`, which can be a boolean value, the name of the terminal
-emulator or the complete command to run the terminal emulator. On Windows,
-only boolean values are valid; if `true` R.nvim will run `Rgui.exe`, and not
-really a terminal emulator.
+`external_term`, which can be `""` (use Neovim's built-in terminal),
+`"default"` (automatically choose a pre-defined terminal), the name of the
+terminal emulator or the complete command to run the terminal emulator. On
+Windows, only boolean values are valid; if `true` R.nvim will run `Rgui.exe`,
+and not really a terminal emulator.
```

Comment:
```suggestion
terminal emulator or the complete command to run the terminal emulator. 
```

I guess we can remove this since it only accept strings now.

Reply:
I'll fix it...

Reply:
I changed the text. I think it's still relevant to say that on Windows we don't run R in an external terminal emulator.